crates/store/re_types/definitions/rerun/archetypes/transform3d.fbs

```diff
@@ -8,21 +8,38 @@ namespace rerun.archetypes;
 
 /// A transform between two 3D spaces, i.e. a pose.
 ///
+/// All components are applied in the order they are listed here.
+/// E.g. if both a 4x4 matrix with a translation and a translation vector are present,
+/// the matrix is applied first, then the translation vector on top.
```

Comment:
We are more and more establishing an alternative pattern of elsewhere of "if X is set, then Y and Z are ignored, with a warning". I still wonder if this might be the better thing to do with mat4x4 and  mat3x3+transform.

Reply:
not a bad point, but the exclusion sets are arguably more complicated than including everything 🤔

Reply:
> exclusion sets are arguably more complicated than including everything 

I agree with this. The new design is conceptually very simple: we have a bunch of different transform components, and we have a well-defined order in which they are applied.